endpoints.json

```diff
@@ -28,7 +28,7 @@
   },
   "GET /api/articles/:article_id": {
     "description": "serves a single article of the given article_id",
-    "queries": [],
+    "queries": ["comment_count"],
```

Comment:
`comment_count` shouldn't be included as a query here

Reply:
Thank you, I missed that one. I've ammended the file now.